helm/defectdojo/templates/pvc.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.django.uwsgi.persistentVolume.enabled -}}
+{{- if not .Values.django.uwsgi.persistentVolume.existingClaim -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  annotations:
+    "helm.sh/resource-policy": keep
+  name: {{ include "defectdojo.fullname" . }}-media
+spec:
+  accessModes:
```

Comment:
Overhere, in case of multiple replicas of django pod, there will be an issue. By this policy you are not able to write read to the same volume from multiple pods.

Reply:
Updated accessMode to "ReadWriteMany".

Reply:
It is not so simple just to convert into RWM, because underlay needs to support it, that is the problem, that is the reason I mentioned NFS, there is no clear solution without StorageClass that supports RWM. I assume for cloud solution like Google or AWS it is already there but for local installation it is responsibility of k8s admin to come up with something.   

@madchap @alles-klar  what do you think?

Reply:
You are right but I believe the usage of storageClass with any kind of internal or external provisioner should be at least an option if not the "default solution".

Reply:
We have to mention that in the docs and provide some documentation but then it should be ok.